src/main/java/org/alliancegenome/curation_api/interfaces/crud/LoggedInPersonCrudInterface.java

```diff
@@ -0,0 +1,28 @@
+package org.alliancegenome.curation_api.interfaces.crud;
+
+
+import java.util.HashMap;
+
+import javax.ws.rs.Consumes;
+import javax.ws.rs.POST;
+import javax.ws.rs.Path;
+import javax.ws.rs.Produces;
+import javax.ws.rs.core.MediaType;
+
+import org.alliancegenome.curation_api.base.interfaces.BaseIdCrudInterface;
+import org.alliancegenome.curation_api.model.entities.LoggedInPerson;
+import org.eclipse.microprofile.openapi.annotations.Operation;
+import org.eclipse.microprofile.openapi.annotations.parameters.RequestBody;
+import org.eclipse.microprofile.openapi.annotations.tags.Tag;
+
+
+@Path("/loggedinperson")
+@Produces(MediaType.APPLICATION_JSON)
+@Consumes(MediaType.APPLICATION_JSON)
+public interface LoggedInPersonCrudInterface {
+    @POST
+    @Path("/savesettings")
+    @Operation(hidden=true)
+    public void saveSettings(@RequestBody HashMap<String, Object> settings);
```

Comment:
@markquintontulloch savesettings doesn't need to be hidden because you have the check on the back end to make sure its the authenticated user.

Reply:
Unhidden